inappmessaging/USERGUIDE.md

```diff
@@ -143,7 +143,7 @@ Create a new class in your project which implements the following file:
 `com.rakuten.tech.mobile.inappmessaging.runtime.UserInfoProvider` <br/>
 Refer to the ECO Demo [example](https://gitpub.rakuten-it.com/projects/ECO/repos/core-android-inappmessaging/browse/test/src/main/java/com/rakuten/test/AppUserInfoProvider.java?at=refs%2Fheads%2Fdev)
 
-This class serves the purpose of providing basic user information such as userId, and RAE token at runtime.
+This class serves the purpose of providing basic user information such as userId, rakutenId, and RAE token at runtime.
 It's best to utilize the [User SDK](https://documents.developers.rakuten.com/android-sdk/user-4.13/).
```

Comment:
Sorry, I didn't notice that there are links to internal pages in the User Guide. These should probably be removed.

Reply:
Removed internal pages